.changeset/angry-spies-approve.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder': patch
+'@backstage/plugin-scaffolder-backend': patch
+---
```

Comment:
With the current implementation, should this be a major? As it's using the OAuth token by default if the OAuth is configured, which would change implementation for any apps using scaffolder + OAuth GitHub configured.

Reply:
Hopefully it shouldn't be a breaking change :+1: